scripts/ci/selective_ci_checks.sh

```diff
@@ -18,12 +18,17 @@
 # shellcheck source=scripts/ci/libraries/_script_init.sh
 . ./scripts/ci/libraries/_script_init.sh
 
+# Parameters:
+# $1 - Merge commit SHA (if pull request - if missing, then all tests are run)
```

Comment:
Can we change this i.e it is not clear.

For example :

```
# $1 - Merge commit SHA (if it is a PR & if SHA is missing, then all tests are run)
```

???

Reply:
Updated the description to be clearer. Simply in 'push' and 'schedule' workflow runs we should run all the tests and in this case we call the scripts without SHA (because there is no Merge SHA in this case and we want to run all tests anyway).